### PR TITLE
updates on samples verbiage

### PR DESCRIPTION
**Pull request by wduraes**
Small changes to the samples verbiage.

### CODE REVIEW

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md [hunk 1]
```diff
@@ -8,14 +8,16 @@ Prerequisites:
...
-- [Azure Command Line Interface utility](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt?view=azure-cli-latest#install-with-one-command) and the IoT extension
-
-  ```shell
-  $ curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
-  $ az extension add --name azure-iot
-  ```
-
-  Alternatively, you can use [azure-iot-explorer](https://github.com/Azure/azure-iot-explorer).
+- Use one of the two:
+    - Graphical user interface to Azure IoT: [azure-iot-explorer](https://github.com/Azure/azure-iot-explorer).
```
Comment (ahsonkhan) [inline sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:12]: Should this directly point to the release/recommended acquisition point?
```suggestion
    - Graphical user interface to Azure IoT: [azure-iot-explorer](https://github.com/Azure/azure-iot-explorer/releases).
```

**Review by ahsonkhan — APPROVED** [review]

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md (hunk 1 above)
Comment (danewalton) [reply sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:12]: no it should just point to the latest

**Review by danewalton — APPROVED** [review]

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md (hunk 1 above)
Comment (ahsonkhan) [reply sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:12]: Point to the latest what?

**Review by ewertons — APPROVED** [review]

File: sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## Introduction
 
+Prerequisites:
```
Comment (momuno) [inline sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md:5]: Why was this added?  There is a Prerequisites section below, and this is the Intro section.

Comment (momuno) [reply sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md:5]: I have reorganized the code to move the prereq section to a header2 level (more noticeable)
